bindgen-tests/tests/expectations/tests/forward-declaration-autoptr.rs

```diff
@@ -20,7 +20,7 @@ impl<T> Default for RefPtr<T> {
     }
 }
 #[repr(C)]
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug)]
```

Comment:
I think losing `Copy, Clone` here may be undesired. It seems that bindgen would also need to manually implement the trait instead of deriving it, since Foo is neither Copy nor Clone